sanic/sanic.py

```diff
@@ -348,9 +348,8 @@ def run(self, host="127.0.0.1", port=8000, debug=False, before_start=None,
         log.debug(self.config.LOGO)
 
         # Serve
-        if ssl is None:
-            proto = "http"
-        else:
+        proto = "http"
```

Comment:
How is this change related to this PR ? What did you want to fix with this change ?